includes/constants/class-payment-method.php

```diff
@@ -39,4 +39,9 @@ class Payment_Method extends Base_Constant {
 		self::CARD_PRESENT,
 		self::INTERAC_PRESENT,
 	];
+
+	const BNPL_METHODS = [
```

Comment:
That's a nice change. Let's change the name to `BNPL_PAYMENT_METHODS` - it sounds a bit better in this specific case.

Reply:
Thank you! I have renamed it. 8275876